### PR TITLE
chapter_3

### PR DESCRIPTION
**Pull request by DaudKhadijah01**

### CODE REVIEW

**Comment by DaudKhadijah01** [comment]
ready

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,8 @@
  *= require_tree .
  *= require_self
  */
+
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/application.css:20]: remove empty line

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.center h1 {
+margin-bottom: 10px;
+}
+
+/* header */
+
+#logo {
+float: left;
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/custom.scss:26]: wrong indent

Comment (ngocvt-0484) [reply app/assets/stylesheets/custom.scss:26]: for example 
image

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Staticpages controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/static_pages.scss:2]: delete it because not use

File: app/helpers/static_pages_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ngocvt-0484) [inline app/helpers/static_pages_helper.rb:1]: remove it because not use

File: app/views/static_pages/help.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<% provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
+<a href="http://www.railstutorial.org/help">Rails Tutorial
```
Comment (ngocvt-0484) [inline app/views/static_pages/help.html.erb:6]: use link_to please

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, "Home" %>
+
+<div class="center jumbotron">
+<h1>Welcome to the Sample App</h1>
```
Comment (ngocvt-0484) [inline app/views/static_pages/home.html.erb:4]: format your html
it is wrong indent

**Comment by ngocvt-0484** [comment]
please delete folder newClone on your sample_app

File: app/helpers/static_pages_helper.rb (hunk 4 above)
Comment (DaudKhadijah01) [reply app/helpers/static_pages_helper.rb:1]: if i remove this, i cannot open my rails

**Comment by DaudKhadijah01** [comment]
ready

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**